examples/backendclient/main.go

```diff
@@ -42,25 +43,50 @@ func bytesToString(data []byte) string {
 }
 
 func ppJSON(s string) {
-	buf := new(bytes.Buffer)
-	json.Indent(buf, []byte(s), "", "  ")
-	log.Println(buf)
+	if verbose {
+		buf := new(bytes.Buffer)
+		json.Indent(buf, []byte(s), "", "  ")
+		log.Println(buf)
+	}
 	return
 }
 
+var (
+	profileName    string
+	filename       string
+	beHost         string
+	bePort         string
+	beCall         string
+	assignment     string
+	delAssignments bool
+	verbose        bool
+)
+
 func main() {
+	flag.StringVar(&filename, "file", "profiles/testprofile.json", "JSON file from which to read match properties")
```

Comment:
look at frontendclient for flag definitions

Reply:
Done.